tests/compiler/panicoverride.nim

```diff
@@ -0,0 +1,12 @@
+{.push stack_trace: off, profiler:off.}
```

Comment:
Please add a short description in a comment, describing the purpose of the file and its contents. Additionally, if you can, add documentation comments to the procedures, describing:
  - Their purpose.
  - When they are called.
  - What assumptions are made by their callers and/or what limitations they should observe (e.g, don't allocate memory, etc.)